cirq/pasqal/pasqal_qubits_test.py

```diff
@@ -219,3 +219,195 @@ def test_to_json():
         'col': 1,
         'lay': 1,
     }
+
+# NEW QUBITS TESTS
```

Comment:
nit: this comment seems unnecessary

Reply:
I agree, it was just a mark for the time the old tests are removed. I have no problem removing it.